EIPS/eip-1011.md

```diff
@@ -235,21 +280,25 @@ Most other decisions were made to minimize changes across clients. For example,
 
 #### Issuance
-A fixed amount of 1M ETH was chosen as `CASPER_BALANCE` to fund the casper contract. This gives the contract enough runway to operate for approximately 2 years (assuming ~10M ETH in validator deposits). Acting similarly to the "difficulty bomb", this "funding crunch" forces the network to hardfork in the relative near future to further fund the contract. This future hardfork is an opportunity to upgrade the contract and transition to full PoS.
+A fixed amount of 1.25M ETH was chosen as `CASPER_BALANCE` to fund the casper contract. This gives the contract enough runway to operate for approximately 2 years (assuming ~10M ETH in validator deposits). Acting similarly to the "difficulty bomb", this "funding crunch" forces the network to hardfork in the relative near future to further fund the contract. This future hardfork is an opportunity to upgrade the contract and transition to full PoS.
 
 The PoW block reward is reduced to 0.6 ETH/block because the security of the chain is greatly shifted from PoW difficulty to PoS finality and because rewards are now issued to both validators and miners. See [here](https://gist.github.com/djrtwo/bc864c0d0a275170183803814b207b9a) for further analysis of the effect of PoW block reward reduction in the context of Hybrid Casper FFG.
 
+In addition to block rewards, miners now receive an issuance reward for including successful `vote` transactions into the block on time. This reward is equal to 1/8th that of the reward the validator receives for the successful `vote`. In total, under optimal FFG conditions, miners receive approximately 1/5th of the total ETH issued by the Casper contract.
+
 Below is a table of deposit sizes with associated annual interest rate and approximate time until funding crunch:
 
-| Deposit Size | Annual Interest | Funding Crunch |
+| Deposit Size | Annual Validator Interest | Funding Crunch |
 | -------- | -------- | -------- |
 | 2.5M ETH | 10.12%   | ~4 years   |
 | 10M ETH  | 5.00%    | ~2 years   |
 | 20M ETH  | 3.52%    | ~1.4 years |
 | 40M ETH  | 2.48%    | ~1 year    |
 
 #### Gas Changes
-Successful casper `vote` transactions are included at the end of the block so that they can be processed in parallel with normal block transactions and cost 0 gas for validators.
+Normal block transactions cannot affect casper `vote` validation results, but casper `vote` validation results can affect normal block transaction execution. Because of this asymmetrical relationship, `vote` transactions can be processed in parallel with normal block transactions if `vote` transactions are placed after all normal block transactions. Because `vote` transactions can be processed in parallel to normal block transactions, `vote` transactions cost 0 gas for validators, ensuring that validators can submit votes even in highly congested or high gas-price periods.
```

Comment:
> casper `vote` validation results can affect normal block transaction execution

Is there an example to illustrate this?

Reply:
a `vote` only modifies very limited state in the contract (vote counts, is_justified, is_finalized, etc), but an external contract could reference the state of the casper contract like `if casper_contract.epoch_10.is_finalized: send_funds()`. the state that `vote` modifies also doesn't change if new deposits or logouts occur because of the way changes are delayed until dynasties change.

- `vote` does not depend on external transactions
- regular transaction can depend on a `vote`
==> put all the votes at the end of execution so the regular transactions can be sure that the casper contract state is stable throughout the block.